cmd/peggo/tx.go

```diff
@@ -1,178 +1,133 @@
-package main
+package peggo
 
 import (
 	"context"
+	"fmt"
+	"os"
 	"time"
 
-	cli "github.com/jawher/mow.cli"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
 	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
-	"github.com/xlab/closer"
-	log "github.com/xlab/suplog"
-
-	"github.com/InjectiveLabs/peggo/orchestrator/cosmos"
-	"github.com/InjectiveLabs/sdk-go/chain/client"
-	"github.com/InjectiveLabs/sdk-go/chain/peggy/types"
+	"github.com/umee-network/peggo/cmd/peggo/client"
+	"github.com/umee-network/peggo/orchestrator/cosmos"
+	peggytypes "github.com/umee-network/umee/x/peggy/types"
```

Comment:
Note to self: I think we need a module in celestia-app for the query types and maybe other bridge related logic. (cc @evan-forbes)